### PR TITLE
Optimize audio_swap, aspect_ratio, overlays

### PR DESCRIPTION
**Pull request by Adib234**
## Summary
- [X] I have read CONTRIBUTING.md to understand how to contribute to this repository :)

<Please summarize what you are trying to achieve, what changes you made, and how they acheive the desired result.>
I'm trying to speed up the following video augmentations: audio_swap, aspect_ratio, and overlays.

Here are the times which I measured with time for i in (seq 5); python -m unittest augly.tests.video_tests.transforms.ffmpeg_test.TransformsVideoUnitTest.test_{respective_augmentation};; end

Runtime with changes for AudioSwap: 3.554s (test finished about 0.5s)
Runtime without changes for AudioSwap: 19.044s (test finished about 13s)

Runtime with changes for AspectRatio: 5.222s (test finished about 2.5s)
Runtime without changes for AspectRatio: 12.486s (test finished about 9s)

Runtime with changes for Overlays: 5.896s (test finished about 2.5s)
Runtime without changes for Overlays: 13.118s (test finished about 9.5s)

## Unit Tests
If your changes touch the `audio` module, please run all of the `audio` tests and paste the output here. Likewise for `image`, `text`, & `video`. If your changes could affect behavior in multiple modules, please run the tests for all potentially affected modules. If you are unsure of which modules might be affected by your changes, please just run all the unit tests.

### Audio
```bash
python -m unittest discover -s augly/tests/audio_tests/ -p "*"
```

### Image
```bash
python -m unittest discover -s augly/tests/image_tests/ -p "*_test.py"
# Or `python -m unittest discover -s augly/tests/image_tests/ -p "*.py"` to run pytorch test too (must install `torchvision` to run)
```

### Text
```bash
python -m unittest discover -s augly/tests/text_tests/ -p "*"
```

### Video
```bash
python -m unittest discover -s augly/tests/video_tests/ -p "*"
```

### All
```bash
python -m unittest discover -s augly/tests/ -p "*"
```

## Other testing

If applicable, test your changes and paste the output here. For example, if your changes affect the requirements/installation, then test installing augly in a fresh conda env, then make sure you are able to import augly & run the unit test

### CODE REVIEW

File: augly/video/augmenters/ffmpeg/audio_swap.py [hunk 1]
```diff
@@ -1,50 +1,61 @@
...
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-i",
+            self.audio_path,
+            "-c:v",
+            "copy",
+            "-af",
+            audio_filters,
```
Comment (jbitton) [inline augly/video/augmenters/ffmpeg/audio_swap.py:53]: have you made sure this command works when videos have no audio stream?

**Review by jbitton — COMMENTED** [review]
overall this lgtm! just have one comment before i can approve

File: augly/video/augmenters/ffmpeg/aspect_ratio.py [hunk 2]
```diff
@@ -39,10 +38,20 @@ def add_augmenter(
...
-            .filter("setdar", **{"ratio": self.aspect_ratio}),
-            {},
-        )
+        command = [
+            "-y",
+            "-i",
+            video_path,
+            "-vf",
+            f"scale=width={new_w}:height={new_h},"
+            + f"pad=width=ceil(iw/2)*2:height=ceil(ih/2)*2,"
```
Comment (zpapakipos) [inline augly/video/augmenters/ffmpeg/aspect_ratio.py:47]:
```suggestion
            + "pad=width=ceil(iw/2)*2:height=ceil(ih/2)*2,"
```

File: augly/video/augmenters/ffmpeg/audio_swap.py [hunk 3]
```diff
@@ -1,50 +1,61 @@
...
+        video_info = get_video_info(video_path)
 
         audio_duration = float(audio_info["duration"])
         audio_sample_rate = float(audio_info["sample_rate"])
 
         start = self.offset
         end = start + float(video_info["duration"])
 
-        audio = ffmpeg.input(self.audio_path).audio
+        audio_filters = f"atrim={start}:{end}," + f"asetpts=PTS-STARTPTS"
```
Comment (zpapakipos) [inline augly/video/augmenters/ffmpeg/audio_swap.py:38]:
```suggestion
        audio_filters = f"atrim={start}:{end}," + "asetpts=PTS-STARTPTS"
```

**Review by zpapakipos — CHANGES_REQUESTED** [review]
nits: f strings don't need to be f strings if no vars are referenced inside

File: augly/video/augmenters/ffmpeg/audio_swap.py (hunk 1 above)
Comment (Adib234) [reply augly/video/augmenters/ffmpeg/audio_swap.py:53]: This command works and I verified this by doing this 
```python
import augly.video.functional as af
import augly.video.helpers.ffmpeg as ah

af.remove_audio('/tmp/test-video-3.mp4','/Users/admin/res.mp4')
ah.extract_audio_to_file('/tmp/test-video-2.mp4','/Users/admin/res1.mp3')
af.audio_swap('/Users/admin/res.mp4','/Users/admin/res1.mp3')
```

**Review by jbitton — APPROVED** [review]
LGTM!

**Pull request closed without merging**